.eslintrc

```diff
@@ -0,0 +1,29 @@
+{
+  "env": {
+    "browser": true,
+    "react-native/react-native": true // whitelists all browser-like globals. https://www.npmjs.com/package/eslint-plugin-react-native
+  },
+  "parser": "@typescript-eslint/parser",
+  "parserOptions": {
+    "ecmaVersion": "latest",
+    "sourceType": "module"
+  },
+  "settings": {
+    "import/resolver": {
+      "typescript": {}
+    }
+  },
+  "plugins": ["react", "react-native", "@typescript-eslint", "prettier"],
+  "extends": [
+    "eslint:recommended",
+    "plugin:react/recommended",
+    "plugin:react/jsx-runtime", // https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/react-in-jsx-scope.md#when-not-to-use-it
+    "plugin:react-hooks/recommended", // https://reactjs.org/docs/hooks-rules.html
+    "plugin:react-native/all",
```

Comment:
@sandrina-p  It's because of react-native eslint rules, I can remove that specific rule though

Reply:
I'm planning to use tailwind so this wouldn't be an issue xD

Reply:
Oh no, please don't 😭 😭 😭 all but tailwind

Reply:
I'd remove that rule yes, as it changes the order of code (including CSS), pls. we can discuss tailwind later on x)

Reply:
Ok, I'll remove it but, regarding tailwind, what are your concerns? I've used [this package](https://github.com/jaredh159/tailwind-react-native-classnames) before and it was pretty nice. I also found [this one](https://www.nativewind.dev/) earlier today and wanted to give it a try as well 😅

Reply:
Fixed in [385210c](https://github.com/sandrina-p/papers-game/pull/15/commits/385210c893f92305280229cd7ef504a94bc3f360)